src/App.tsx

```diff
@@ -1,9 +1,20 @@
 import React from 'react';
+import styled, { ThemeProvider } from 'styled-components';
+import theme from './theme/theme'
+import GlobalStyles from './theme/GlobalStyles'
+
+const Heading = styled.h1`
```

Comment:
Even though this was set up for the test, I would like to make note that this styled components called `Heading` will be more semantic/descriptive in a future PR.